rust/azure_iot_operations_connector/examples/base_connector_sample.rs

```diff
@@ -161,7 +162,7 @@ async fn run_dataset(dataset_client: DatasetClient) {
     }
     let mut count = 0;
     loop {
-        tokio::time::sleep(Duration::from_secs(5)).await;
+        tokio::time::sleep(Duration::from_secs(10)).await;
```

Comment:
any reason for increasing sleep ?

Reply:
just because it runs constantly and there isn't a way to easily pause execution of the pod (that I know about at least), I wanted it to execute a little bit less while still making it easy to validate that things are working